.circleci/config.yml

```diff
@@ -55,6 +55,9 @@ jobs:
       - run:
           name: Test Code
           command: docker run app:build test
+      - run:
+          name: Lint Code
+          command: docker run app:build flake8 leanplum_data_export/leanplum_data_export/ leanplum_data_export/tests/ --max-line-length 100
```

Comment:
nit: `--max-line-length 100` could be moved to some project config file so it's picked up by IDEs.

Reply:
That's a good idea. I'll change it